mlflow/server/js/src/experiment-tracking/components/ExperimentListView.tsx

```diff
@@ -126,18 +136,25 @@ export const ExperimentListView = ({ experiments, error }: Props) => {
         <TableFilterLayout>
           <TableFilterInput
             data-testid="search-experiment-input"
-            placeholder="Search experiments by name"
+            placeholder={intl.formatMessage({
+              defaultMessage: 'Filter experiments by name, tags or attributes',
+              description: 'Placeholder text inside experiments search bar',
+            })}
             componentId="mlflow.experiment_list_view.search"
-            value={searchInput}
+            defaultValue={searchFilter}
             onChange={handleSearchInputChange}
+            onSubmit={handleSearchSubmit}
```

Comment:
is it possible to only update the real search filter `onPressEnter`? i worry about sending a network request on every keypress, and enter key is currently the trigger for most other search bars

```suggestion
            onPressEnter={(e) => e.target.value}
```

Reply:
ahh sorry! i see this is handled already